app/forms/webauthn_verification_form.rb

```diff
@@ -127,13 +132,42 @@ def generic_error_message
     end
   end
 
-  def extra_analytics_attributes
-    auth_method = if webauthn_configuration&.platform_authenticator
-                    'webauthn_platform'
-                  else
-                    'webauthn'
-                  end
+  def screen_lock_error_message
+    if user_has_other_authentication_method?
+      t(
+        'two_factor_authentication.webauthn_error.screen_lock_other_mfa',
+        link: link_to(
```

Comment:
this probably should have `_html` on it? the only exception is when strings get written to `flash`?

Reply:
The test got angry at me about it. I tested without the `_html` and it seemed to work fine for me.

I think the alternative would be to apply `_html` to both the "parent" and "child" strings like what's done above with `connect_html`, but I'd opted to avoid it since it seemed to work without.

Reply:
what's rendering this error message?

Reply:
> what's rendering this error message?

I'm not sure what you mean by this. Do you mean how to reproduce the error message being shown?

Reply:
I meant, where is the code that renders the error in a template.

I did some spelunking and found that the error gets written to the flash:
https://github.com/18F/identity-idp/blob/bf8a54908edde91ec1f8bd709908d17e73137fdd/app/controllers/two_factor_authentication/webauthn_verification_controller.rb#L58

which explains why it renders correctly without being named `_html` because all flash messages get `.html_safe` 😭 

https://github.com/18F/identity-idp/blob/bf8a54908edde91ec1f8bd709908d17e73137fdd/app/components/flash_component.html.erb#L2

Reply:
😬  I'm not sure I'd realized that. I'd be interested in seeing what it might take to move away from that, though definitely a separate effort. I'll see if it also works to just add `_html` to both the "parent" and "child" and that would be future-proof at least.

Reply:
Added the suffix in 05bc0f059 . I did a little bit of testing of what it'd look like to remove `html_safe` to verify that it'd work, but it looks like something is dropping the HTML safety of the suffixed string somewhere along the way. Something to look at later if / when we try to drop the auto-safety.